src/main/java/edu/ucsb/nceas/mdqengine/Controller.java

```diff
@@ -49,16 +57,22 @@ public class Controller {
     private static com.rabbitmq.client.Connection RabbitMQconnection;
     private static com.rabbitmq.client.Channel RabbitMQchannel;
 
-    // Default values for the RabbitMQ message broker server. The value of 'localhost' is valid for
-    // a RabbitMQ server running on a 'bare metal' server, inside a VM, or within a Kubernetes
-    // where metadig-controller and the RabbitMQ server are running in containers that belong
-    // to the same Pod. These defaults will be used if the properties file cannot be read.
+    // Default values for the RabbitMQ message broker server. The value of
+    // 'localhost' is valid for
+    // a RabbitMQ server running on a 'bare metal' server, inside a VM, or within a
+    // Kubernetes
+    // where metadig-controller and the RabbitMQ server are running in containers
+    // that belong
+    // to the same Pod. These defaults will be used if the properties file cannot be
+    // read.
     // These values are read from a config file, see class 'MDQconfig'
     private static Boolean bookkeeperEnabled = false;
     private static String RabbitMQhost = null;
     private static int RabbitMQport = 0;
     private static String RabbitMQpassword = null;
     private static String RabbitMQusername = null;
+    private static String monitorSchedule = null;
+    private static boolean monitor = true;
```

Comment:
I often think internal comments explaining what a state variable is used for can be helpful to explaining the architecture of an app.